avocado_vt/conf.d/vt.conf

```diff
@@ -4,7 +4,11 @@ backup_image_before_test = True
 # Restore image after testing (if backup present)
 restore_image_after_test = True
 # Keep guest running between tests (faster, but unsafe)
-keep_guest_running=False
+keep_guest_running = False
+# Make the temporary dir path persistent across jobs if needed.
+# By default the data in the temporary directory will be wiped oafter each test
+# in some cases and after each job in others.
+persistent_tmp_dir =
```

Comment:
Would you please elaborate why the option is put in `vt.setup` instead of `vt.common`?

Reply:
Because this is where most options related to setup between test is, e.g.:
```
# Keep guest running between tests (faster, but unsafe)
keep_guest_running = False
```
or

```
# Backup image before testing (if not already backed up)
backup_image_before_test = True
# Restore image after testing (if backup present)
restore_image_after_test = True
```

but if you mean to rename it as general purpose `tmp_dir` then it would make more sense in the `vt.common` section. I will do this in yet another push.